config/locales/client.en.yml

```diff
@@ -115,6 +115,9 @@ en:
       composer:
         toggle_toolbar: "Toggle toolbar"
 
+      quote:
+        original_channel: "Originally posted in %{channel}."
```

Comment:
For terminology's sake: Do we use "post" a chat message? Just thought "post" may imply it's a post. (If you care, "sent" may be less conflicting)

Reply:
@nattsw haha you know I just thought that as I was reading the screenshot I posted :joy: "Originally sent" is probably better I will use that :+1: